packages/babel-helper-wrap-function/src/index.js

```diff
@@ -3,7 +3,7 @@ import nameFunction from "@babel/helper-function-name";
 import template from "@babel/template";
 import * as t from "@babel/types";
 
-const buildWrapper = template(`
+const buildWrapper = template.expression(`
   (() => {
```

Comment:
It might be better to use a plain function instead of arrow function in this case, since you do not depend on `this`. It should have a (negligibly) smaller runtime cost, as well as avoiding the big code size cost when arrow functions are transformed in `spec` mode.